recipes/sqlite/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version="3.11.1" %}
+{% set version_split=version.split(".") %}
+{% set major=version_split[0] %}
+{% set minor=version_split[1]|int * 100 %}
+{% set bugfix=version_split[2]|int * 100 %}
+{% set version_coded=(major ~ (minor|string)[:3] ~ (bugfix|string)[:3]) %}
+
+package:
+  name: sqlite
+  version: {{ version }}
+
+source:
+  fn: sqlite-autoconf-{{ version_coded }}.tar.gz
+  url: https://www.sqlite.org/2016/sqlite-autoconf-{{ version_coded }}.tar.gz
+  sha1: c4b4dcd735a4daf5a2e2bb90f374484c8d4dad29
+
+build:
+  number: 0
```

Comment:
Yes, I think so.  I copied the bld.bat from earlier work of mine, but copied meta.yaml from @jjhelmus.  I'll add it in.